### PR TITLE
Wire up login page

### LINKED ISSUE #8: Make login page
- [x] Wire up login api
- [x] Cleanup ui

### PR DESCRIPTION
**Pull request by mdwagner**
NOTE: after merge, will need to run `docker pull mdwagner/next-great-idea:latest` once latest is updated. this is because I added `apollo` to dev-container image.
NOTE 2: please run `npm run migrate` in hasura directory.

### CODE REVIEW

**Comment by mdwagner** [comment]
@wtmerchant3 this is pretty much ready to go, but we can wait until next conversation before we merge it

**Review by wtmerchant3 — APPROVED** [review]

**Pull request merged**
